scan.go

```diff
@@ -118,6 +122,8 @@ func newScanCommand() *cobra.Command {
 		"Analyze transitive dependencies")
 	cmd.Flags().IntVarP(&transitiveDepth, "transitive-depth", "", 2,
 		"Analyze transitive dependencies till depth")
+	cmd.Flags().BoolVarP(&dependencyUsageEvidence, "dependency-usage-evidence", "", false, "Enable dependency usage evidence during scan")
```

Comment:
Should we name code related config flags as `--code-dependency-usage-evidence`? Just to make them `intention revealing` that they are associated with code analysis?

Also I think `dependency-usage-evidence` should be `true` by default because its a fairly low cost (CPU/memory) and reliable analysis.